### PR TITLE
PP-6741 - Standardises formatting of all API parameter placeholders

### PR DESCRIPTION
**Pull request by NathanD-GDS**
### Context
Previously, there were 5 different formats for parameter placeholders in the Pay tech docs. This PR standardises the formatting of these.

### Changes proposed in this pull request

- Standardises API parameter placeholders to `{CAPITAL_LETTERS}` format.
- Adds additional clarification after placeholders with 'Replace `{PLACEHOLDER_NAME}` with...' sentences.

#### Pre-i'er / 2i'er:

- Do the clarification sentences ('Replace `{PLACEHOLDER_NAME}` with...') read okay? The wording on some of them was tricky.

#### Devs:

- Do the examples still make sense? Especially the query parameters placeholder in _Refund a payment_ -> _Searching refunds_ 
- Have I broken anything?

### CODE REVIEW

File: source/refunding_payments/index.html.md.erb [hunk 1]
```diff
@@ -252,7 +261,7 @@ Example search response:
     {
       "refund_id": "efj6834secviyyxe8vs861pg3j",
       ...
-    }    
+    }
```
Comment (NathanD-GDS) [inline source/refunding_payments/index.html.md.erb:264]: I don't know what this is, how I did it, how to undo it, or whether it even makes a difference O_O

Comment (rorymalcolm) [reply source/refunding_payments/index.html.md.erb:264]: Your editor removed whitespace, there were a load of spaces after that `}`, this is fine

File: source/api_reference/index.html.md.erb [hunk 2]
```diff
@@ -42,7 +42,7 @@ When you make an API call, you need to use an “Authorization” HTTP header to
 provide your API key, with a “Bearer” prefix. For example:
 
 ```text
-Authorization: Bearer <YOUR-API-KEY-HERE>
+Authorization: Bearer {YOUR-API-KEY}
```
Comment (jonathanglassman) [inline source/api_reference/index.html.md.erb:45]: Should this be using underscores rather than hyphens?

File: source/making_payments/index.html.md.erb [hunk 3]
```diff
@@ -286,15 +286,19 @@ Your user can select a link on the payment page to cancel a payment that’s in
...
+`POST /v1/payments/{PAYMENT_ID}/cancel`
+
+Replace `{PAYMENT_ID}` with the ID of the payment that you are cancelling.
 
 A `204` response indicates success. Any other response indicates an error.
 
-### Find out if you can cancel a payment
+### Check if you can cancel a payment
+
+Use a GET request against a single payment to check if you can cancel it:
```
Comment (jonathanglassman) [inline source/making_payments/index.html.md.erb:297]: General point - minimise "it", "that" and "they" to minimise ambiguity. I would rephrase to "...cancel that payment." This is not a huge issue given that there are only 2 entities in the sentence, the GET request and the single payment, but it's a good habit to get into.

Also, should the GET be in `` i.e. `GET`?

File: source/delayed_capture/index.html.md.erb [hunk 4]
```diff
@@ -47,14 +47,16 @@ payment confirmation email to your user once we've received and processed your s
 If a payment is available for capture, you can see its capture URL in
 responses to API calls. For example:
 
-* `GET /v1/payments/{paymentID}`
 * `GET /v1/payments`
+* `GET /v1/payments/{PAYMENT_ID}`
 
-The `"__links"` object will contain:`
+Replace ``{PAYMENT_ID}`` with the ID of the payment that you want to see the capture URL for.
```
Comment (cmcnallygds) [inline source/delayed_capture/index.html.md.erb:53]:
```suggestion
Replace ``{PAYMENT_ID}`` with the ID of the payment you want to see the capture URL for.
```

File: source/making_payments/index.html.md.erb [hunk 5]
```diff
@@ -208,7 +208,7 @@ You can use the `href` and `chargeTokenId` to make a `POST` API request to `next
 
 #### provider_id
 
-`provider_id` is the unique ID that your PSP generated for this payment. 
+`provider_id` is the unique ID that your PSP generated for this payment.
```
Comment (cmcnallygds) [inline source/making_payments/index.html.md.erb:211]:
```suggestion
`provider_id` is the unique ID your PSP generated for this payment.
```

File: source/making_payments/index.html.md.erb [hunk 6]
```diff
@@ -286,15 +286,19 @@ Your user can select a link on the payment page to cancel a payment that’s in
 progress. Alternatively, a service can make the following API call for a given
 `paymentId`:
 
-`POST /v1/payments/paymentId/cancel`
+`POST /v1/payments/{PAYMENT_ID}/cancel`
+
+Replace `{PAYMENT_ID}` with the ID of the payment that you are cancelling.
```
Comment (cmcnallygds) [inline source/making_payments/index.html.md.erb:291]:
```suggestion
Replace `{PAYMENT_ID}` with the ID of the payment you are cancelling.
```

File: source/making_payments/index.html.md.erb [hunk 7]
```diff
@@ -286,15 +286,19 @@ Your user can select a link on the payment page to cancel a payment that’s in
...
-### Find out if you can cancel a payment
+### Check if you can cancel a payment
+
+Use a `GET` request against a single payment to check if you can cancel that payment:
 
-Use a GET request against a single payment to find out if you can cancel it:
+`GET /v1/payments/{PAYMENT_ID}`
 
-`GET /v1/payments/paymentId`
+Replace `{PAYMENT_ID}` with the ID of the payment that you are checking.
```
Comment (cmcnallygds) [inline source/making_payments/index.html.md.erb:301]:
```suggestion
Replace `{PAYMENT_ID}` with the ID of the payment you are checking.
```

File: source/refunding_payments/index.html.md.erb [hunk 8]
```diff
@@ -56,7 +56,9 @@ For a completed £90 payment where £30 has already been refunded:
 
 ## Creating a refund
 
-`POST /v1/payments/PAYMENT-ID/refunds`
+`POST /v1/payments/{PAYMENT_ID}/refunds`
+
+Replace `{PAYMENT_ID}` with the ID of the payment that you are refunding.
```
Comment (cmcnallygds) [inline source/refunding_payments/index.html.md.erb:61]:
```suggestion
Replace `{PAYMENT_ID}` with the ID of the payment you are refunding.
```

File: source/refunding_payments/index.html.md.erb [hunk 9]
```diff
@@ -110,7 +112,12 @@ Use the `status` field to [check the status of a refund](#checking-the-status-of
 
 ## Checking the status of a refund
 
-`GET /v1/payments/PAYMENT-ID/refunds/REFUND-ID`
+`GET /v1/payments/{PAYMENT_ID}/refunds/{REFUND_ID}`
+
+Replace:
+
+* `{PAYMENT_ID}` with the ID of the payment that you are checking
```
Comment (cmcnallygds) [inline source/refunding_payments/index.html.md.erb:119]:
```suggestion
* `{PAYMENT_ID}` with the ID of the payment you are checking
```

File: source/refunding_payments/index.html.md.erb [hunk 10]
```diff
@@ -110,7 +112,12 @@ Use the `status` field to [check the status of a refund](#checking-the-status-of
...
+* `{REFUND_ID}` with the ID of the refund that you are checking
```
Comment (cmcnallygds) [inline source/refunding_payments/index.html.md.erb:120]:
```suggestion
* `{REFUND_ID}` with the ID of the refund you are checking
```

File: source/reporting/index.html.md.erb [hunk 11]
```diff
@@ -269,7 +271,9 @@ You can also sign in to the [GOV.UK Pay admin tool](https://selfservice.payments
 
 Use the following API call to get a list of a payment’s events:
 
-`GET /v1/payments/{paymentId}/events`
+`GET /v1/payments/{PAYMENT_ID}/events`
+
+Replace `{PAYMENT_ID}` with the ID of the payment that you are getting the events of.
```
Comment (cmcnallygds) [inline source/reporting/index.html.md.erb:276]:
```suggestion
Replace `{PAYMENT_ID}` with the ID of the payment you are getting the events of.
```

**Review by kbottla — APPROVED** [review]
